### ISSUE TITLE
MOT17 Exploding Training Gradients

### ISSUE THREAD

**Issue by taheranjary** [SysCV/qdtrack#81]
Hi there,
Awesome results you guys.

When I train QDTrack on MOT17 dataset, I notice the training classification losses to be very large, and eventually they become NaN. Any ideas as to why this is happening?

I use Pytorch 1.7.1 with CUDA 11.0

**Comment by tanjary21** [comment]
Hi there.

If we train the model for long enough, the losses stabilize. But the MOTA and IDF1 etc scores are all NaNs. Why is this?

**Comment by OceanPang** [comment]
@taheranjary We use [this gradient clip](https://github.com/SysCV/qdtrack/blob/b28af06c7fdb6ce99b967302c0c7e9a557d508bf/configs/mot17/qdtrack-frcnn_r50_fpn_4e_mot17.py#L97) to stabilize the training. 

@tanjary21 Can you show your log here?

**Comment by tanjary21** [comment]
Hi there,

I found the problem. The path to your trained model was not defined. It was essentially training from scratch and not using your pre-trained model.

Thank you anyways :)
